pkg/entities/nfts.go

```diff
@@ -47,18 +47,20 @@ var (
 func (e *Entity) GetNFTDetail(symbol, tokenID string) (*response.IndexerNFTToken, error) {
 	// get collection
 	collection, err := e.repo.NFTCollection.GetBySymbol(symbol)
+	// cannot find collection in db
 	if err != nil {
 		err = fmt.Errorf("failed to get nft collection : %v", err)
 		return nil, err
 	}
 	data, err := e.indexer.GetNFTDetail(collection.Address, tokenID)
+	// cannot find collection in indexer
```

Comment:
handle error indexer return "record not found"
![image](https://user-images.githubusercontent.com/39881166/175892256-a3e92237-9762-42ad-8671-c3eb27139cfb.png)